packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -90,8 +93,9 @@ interface ImageDataMetaData {
 }
 // TODO This needs to be exposed as its published to consumers.
 type CalibrationEvent = {
-  rowScale: number;
-  columnScale: number;
+  // The scale is the image pixel distance divisor to create unit values
+  scale: number;
```

Comment:
Note that the scaling to mm conversion has nothing to do with how the image is displayed now - that means you can correctly display it at various aspect ratios independently of the pixel spacing.  This is required for odd-aspect ratio images such as fractures where an angled view is taken.